### PR TITLE
Capitalize the first letter of string in JavaScript

### PR DESCRIPTION
**Pull request by crisbnk**
Merge to Master

### CODE REVIEW

**Comment by Rafase282** [comment]
Seems good to me. Anyone else care to check and let me know.

**Comment by Rafase282** [comment]
@crisbnk There is only one change needed, I sent a pull request to show you what it is, you can either merge it on your end or edit this one and add it. https://github.com/crisbnk/wiki/pull/1

**Comment by Rafase282** [comment]
All that is left is to squash.

File: Capitalize-First-Letter-Of-String.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+var string = "freecodecamp";
+
+function capitalizeFirstLetter(str) {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+capitalizeFirstLetter(string); // Returns "Freecodecamp"
+```
+
+Or you should add that function to the `String.prototype` and chain it with other methods.
```
Comment (abhisekp) [inline Capitalize-First-Letter-Of-String.md]: _you may add that function_

_which is not a good practice_

but to add it to `String.prototype`, use the following code

```js
Object.defineProperty(String.prototype, 'capitalizeFirstLetter', {
    get() {
        return this.charAt(0).toUpperCase() + this.slice(1);
    }
})
```

File: Capitalize-First-Letter-Of-String.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+```javascript
+var string = "freecodecamp";
+
+String.prototype.capitalizeFirstLetter = function() {
+  return this.charAt(0).toUpperCase() + str.slice(1);
```
Comment (abhisekp) [inline Capitalize-First-Letter-Of-String.md]: `str.slice(1)` should be `this.slice(1)`

**Comment by crisbnk** [comment]
@abhisekp I've implemented the suggestions. Let me know what do you think.

**Comment by Rafase282** [comment]
I Squashed, check #478

**Pull request closed without merging**